components/SearchAutocomplete.tsx

```diff
@@ -24,35 +24,32 @@ import {
 import useOutsideClickHandler from 'hooks/useOutsideClickHandler'
 import Group, { GroupProps } from './Group'
 
-export interface SearchOptionProps {
+export interface SearchOptionGroupType<OptionType> {
   label: string
-  value: string
+  options: OptionType[]
 }
```

Comment:
Was this generic change prompted by some warning / error? I'm curious what `OptionType` represents currently, and whether we could make that a discrete named type rather than a generic?